contracts/colony/ColonyAuthority.sol

```diff
@@ -85,8 +85,10 @@ contract ColonyAuthority is CommonAuthority {
     addRoleCapability(ARBITRATION_ROLE, "setExpenditurePayoutModifier(uint256,uint256,uint256,uint256,int256)");
     addRoleCapability(ARBITRATION_ROLE, "setExpenditureClaimDelay(uint256,uint256,uint256,uint256,uint256)");
 
-    // Added in colony v5
+    // Added in colony v5 (current version)
     addRoleCapability(ARBITRATION_ROLE, "transferStake(uint256,uint256,address,address,uint256,uint256,address)");
+    addRoleCapability(ARBITRATION_ROLE, "emitDomainReputationPenalty(uint256,uint256,uint256,address,int256)");
+    addRoleCapability(ARBITRATION_ROLE, "emitSkillReputationPenalty(uint256,address,int256)");
```

Comment:
I've been thinking it would actually be easier to go through and do the `finishUpgrade` function in a separate PR right before release. Otherwise every other rebase (it seems like) requires going through and sorting out merge errors in a pretty error-prone way. Doing it once off of all the changes in `ColonyAuthority` seems like a better approach.

Reply:
I have a powerful aversion to code being merged in to `master` knowing that it needs more work to be released.

Reply:
Mkay, I can do it but it's just a lot of unnecessary merge conflicts down the road.

Reply:
Surely the merge conflicts are trivial to resolve? Two commits have added to the function, and you just include all the calls to `colonyAuthority.setRoleCapability` from both? If we had two complicated inclusions to `finishUpgrade`, maybe they'd be slightly harder, but even in those cases I'd expect very few such commits to actually interact with each other.

Reply:
There's probably a good way to do it, but last time we had the function signatures enumerated as constants separately (i.e. `SIG1`, SIG2`, etc), which made merging multiple branches with different additions very clunky, since we'd have to re-assign numbers to all the function signatures. A different approach which doesn't allow for these conflicts would make things easier.